### PR TITLE
Fix documentation typos

### PR DESCRIPTION
**Pull request by AdvancedUno**
### Description

Documentation update to fix typos.

### Motivation and Context

- This pull request fixes typos and adds missing period in following pages.
   - https://onnxruntime.ai/docs/build/inferencing.html
   - https://onnxruntime.ai/docs/build/training.html
   - https://onnxruntime.ai/docs/build/eps.html

### CODE REVIEW

**Comment by AdvancedUno** [comment]
@microsoft-github-policy-service agree

File: docs/build/eps.md [hunk 1]
```diff
@@ -776,7 +776,7 @@ Linux example:
 
 See more information on the CANN Execution Provider [here](../execution-providers/community-maintained/CANN-ExecutionProvider.md).
 
-### Prerequisites
+### Prerequisitesß
 {: .no_toc }
```
Comment (wschin) [inline docs/build/eps.md:780]: What is `beta`?

Comment (AdvancedUno) [reply docs/build/eps.md:780]: Oh thanks, that is my typo.... 😅

**Comment by justinchuby** [comment]
please remove binary files

**Comment by AdvancedUno** [comment]
@justinchuby Hi! Did I add binary files?

**Comment by justinchuby** [comment]
.DS_Store seen in files changed

**Comment by AdvancedUno** [comment]
@justinchuby Oh Thanks!

**Review by natke — APPROVED** [review]

**Pull request merged**